submissions/Zakkarat/html-forms/index.html

```diff
@@ -0,0 +1,460 @@
+            </div>
+          </section>
+          <section class="documentUpload">
+            <div class="allTextDocumentUpload">
+              <span class="numberSection"><b>1</b></span>
+              <h2 class="uploadText">Upload your documents</h2>
+              <h5 class="smallUploadText">(Doc, Docx, Text, PDF, Markdown)</h5>
+            </div>
+            <label tabindex="0" for="file" class="uploadFile"
+              ><input
+                type="file"
+                id="file"
+                name="file"
+                value="files"
+                class="inputHideFile"
+              /><span class="upload"></span
+              ><span class="inputFileText"
+                ><b>Upload your files</b></span
+              ></label
+            >
+            <table class="wideTable">
+              <tr>
+                <th class="th-to-left">FILE NAME</th>
+                <th class="th-to-left">FILE SIZE</th>
+                <th class="th-to-left">NUMBER OF WORDS</th>
+                <th class="th-to-left">REMOVE</th>
+              </tr>
+              <tr>
+                <td class="image-doc">buisness_operations.docx</td>
+                <td>78 kB</td>
+                <td>3894 words</td>
+                <td class="td-to-center">
+                  <img
+                    tabindex="0"
+                    src="./img/waste-bin.svg"
+                    class="deleteFile"
+                  />
+                </td>
+              </tr>
+              <tr>
+                <td>buisness_financial.docx</td>
+                <td>23 kB</td>
+                <td>2123 words</td>
+                <td class="td-to-center">
+                  <img
+                    tabindex="0"
+                    src="./img/waste-bin.svg"
+                    class="deleteFile"
+                  />
+                </td>
+              </tr>
+            </table>
+          </section>
+          <section class="documentLanguage">
+            <div class="allTextDocumentUpload">
+              <span class="numberSection"><b>2</b></span>
+              <h2 class="uploadText">Choose your languages pairings</h2>
+            </div>
+            <div class="block-to-center">
+              <div class="select-button select-button-from">
+                <select name="language" class="select-button-inner" required>
+                  <option value="english" selected>English</option>
+                  <option value="Arabian">Arabian</option>
+                  <option value="Kudrish">Kudrish</option>
+                  <option value="Ukrainian">Ukrainian</option>
+                </select>
+              </div>
+            </div>
+            <div class="block-to-center">
+              <div class="select-button select-button-to">
+                <select name="language" class="select-button-inner" required>
+                  <option value="english">English</option>
+                  <option value="Arabian" selected>Arabian</option>
```

Comment:
Use `camelCase` for value

Reply:
Sorry, do you mean I should write "arabian", not "Arabian"?

Reply:
@Zakkarat yes, it will be more safe

Reply:
@Zakkarat yes